force-app/main/default/lwc/listPage/listPage.js

```diff
@@ -0,0 +1,191 @@
+import { LightningElement, wire, track } from 'lwc';
+import { refreshApex } from '@salesforce/apex';
+import { loadStyle } from 'lightning/platformResourceLoader';
+import { updateRecord } from 'lightning/uiRecordApi';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import getAllLeads from '@salesforce/apex/LeadController.getAllLeads';
+import LEAD_OBJECT from '@salesforce/schema/Lead';
+import NAME_FIELD from '@salesforce/schema/Lead.Name';
+import PHONE_FIELD from '@salesforce/schema/Lead.Phone';
+import EMAIL_FIELD from '@salesforce/schema/Lead.Email';
+import STATUS_FIELD from '@salesforce/schema/Lead.Status';
+import { getPicklistValues, getObjectInfo } from 'lightning/uiObjectInfoApi';
+
+const COLS = [
+  {
+    label: 'Name',
+    fieldName: NAME_FIELD.fieldApiName,
+    type: 'text',
+    sortable: true,
+    editable: false,
+  },
+  {
+    label: 'Phone',
+    fieldName: PHONE_FIELD.fieldApiName,
+    type: 'phone',
+    sortable: true,
+    editable: true,
+  },
+  {
+    label: 'Email',
+    fieldName: EMAIL_FIELD.fieldApiName,
+    type: 'email',
+    sortable: true,
+    editable: true,
+  },
+  {
+    label: 'Lead Status',
+    fieldName: 'Status',
+    type: 'picklistColumn',
+    editable: true,
+    typeAttributes: {
+      placeholder: 'Edit Status',
+      options: { fieldName: 'pickListOptions' },
+      value: { fieldName: 'Type' },
+      context: { fieldName: 'Id' },
+    },
+  },
+  {
+    label: 'Owner Alias',
+    fieldName: 'OwnerAlias',
+    type: 'text',
+    sortable: true,
+    editable: false,
+  },
+];
+
+export default class listPage extends LightningElement {
+  @track columns = COLS;
+  @track draftValues = [];
+  @track visibleRecords = [];
+  @track allLeads = [];
+  @track error;
+  @track isLoading = true;
+  @track dropdownVisible = false;
+  @track pickListOptions;
+  wiredLeadsResult;
+
+  statusOptions = [
+    { label: 'New', value: 'New' },
+    { label: 'No Contact', value: 'No Contact' },
+    { label: 'Contacted', value: 'Contacted' },
+    { label: 'Qualified', value: 'Qualified' },
+    { label: 'Not Qualified-Closed', value: 'Not Qualified-Closed' },
+  ];
+
+  @wire(getObjectInfo, { objectApiName: LEAD_OBJECT })
+  objectInfo;
```

Comment:
why we are using two wire method

Reply:
It was required because its fetching two different information and to make it dynamic .

Reply:
is it not possible using connectcall back?